src/libraries/System.Private.CoreLib/src/System/MemoryExtensions.cs

```diff
@@ -617,40 +666,9 @@ public static ReadOnlyMemory<char> AsMemory(this string? text, Range range)
                         Unsafe.As<T, short>(ref value3),
                         span.Length);
                 }
-                else if (Unsafe.SizeOf<T>() == sizeof(int))
-                {
-                    return SpanHelpers.IndexOfAnyExceptValueType(
-                        ref Unsafe.As<T, int>(ref MemoryMarshal.GetReference(span)),
-                        Unsafe.As<T, int>(ref value0),
-                        Unsafe.As<T, int>(ref value1),
-                        Unsafe.As<T, int>(ref value2),
-                        Unsafe.As<T, int>(ref value3),
-                        span.Length);
-                }
-                else if (Unsafe.SizeOf<T>() == sizeof(long))
-                {
-                    return SpanHelpers.IndexOfAnyExceptValueType(
-                        ref Unsafe.As<T, long>(ref MemoryMarshal.GetReference(span)),
-                        Unsafe.As<T, long>(ref value0),
-                        Unsafe.As<T, long>(ref value1),
-                        Unsafe.As<T, long>(ref value2),
-                        Unsafe.As<T, long>(ref value3),
-                        span.Length);
-                }
```

Comment:
Why did this lose its int and long vectorization?

Reply:
I see.  We're keeping the XxExcept variants consistent with what the Xx counterparts already had.  Ok.